app.js

```diff
@@ -50,7 +65,16 @@ console.log(
 //    that checks if the year is a leap year, and returns the correct
 //    number of days for Feb. The default value for the year should be
 //    the current year
-function daysInMonth() {}
+function daysInMonth(month) {
+    if (month == 2) {
```

Comment:
in this function, you'll want to use what is called 'deep equals' which is === instead of ==